lib/sidewalks/informants/twitter.rb

```diff
@@ -0,0 +1,34 @@
+module Sidewalks
+  module Informants
+    class Twitter
+
+      attr_accessor(
+        :client,
+        :consumer_key,
+        :consumer_secret,
+        :access_token,
+        :access_token_secret
+      )
+
+      def self.client(config: {})
+        new(config).client
+      end
+
+      def initialize(config: {})
+        @consumer_key = config[:consumer_key] || ENV['TWITTER_CONSUMER_KEY']
+        @consumer_secret = config[:consumer_secret] || ENV['TWITTER_CONSUMER_SECRET']
+        @access_token = config[:access_token] || ENV['TWITTER_OAUTH_ACCESS_TOKEN']
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.